.github/workflows/e2e-deploy.yml

```diff
@@ -9,6 +9,7 @@ on:
     branches:
       - main
 
+concurrency: ${{ github.workflow }}-${{ github.ref }}
```

Comment:
Wouldn't we need the same concurrency group between `main` and any PR branch to prevent concurrent workflow runs? In the case where the cron workflow is running on `main` at the same time a PR is opened from a branch called `feature-branch` I think we'd get different groups here. The PR's concurrency group would be `"Test e2e Deploy to WP Engine-feature-branch` while the cron group would be `"Test e2e Deploy to WP Engine-main`. Do I have that right?

I'm also wondering whether we should throw `cancel-in-progress: false` in there to be safe. The [GitHub docs](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs) seem to contradict themselves on the default behavior 🤔 

> To also cancel any currently running job or workflow in the same concurrency group, specify cancel-in-progress: true.

And then later describing default behavior...

> When you define a concurrency key, GitHub Actions ensures that only one workflow or job with that key runs at any given time. If a new workflow run or job starts with the same concurrency key, GitHub Actions will cancel any workflow or job already running with that key.

Reply:
That's a great catch! I just biffed using a hardcoded value instead of the ref, but it is fixed now.

Also, I had left off `cancel-in-progress` to use the default, but it doesn't hurt anything to add it. If nothing else, it is nice documentation of intent 😄

Both of these callouts were addressed with a rebase just a few minutes ago.

Reply:
Nice! That should get us a single concurrency group shared across the workflow regardless of which branch initiated it. 

What are your thoughts on dropping `-main` from the end? To me, seeing `-main` implies the concurrency group is somehow limited to the `main` branch when it's not.

Reply:
That's an interesting point! While the cron is not limited to `main`, we are triggering on pushes and PRs that target `main`.

Do you think it would be more correct to simply label dynamically with the workflow name?

Reply:
Yeah, just the workflow name makes sense to me.

I see what you mean with the target always being `main` - that's a good point in favor of keeping it. I may have been skewed by `github.ref` since we were just looking at that and it's populated with the source branch rather than the target.

In any case, it's a nit. I'm good with either way!